src/core/src/op/ceiling.cpp

```diff
@@ -62,7 +62,6 @@ bool Ceiling::has_evaluate() const {
     case element::u16:
     case element::u32:
     case element::u64:
-    case element::f16:
```

Comment:
It cannot be removed as will cause the GPU performance drop.  See ticket [CVS-121845](https://jira.devtools.intel.com/browse/CVS-121845)